frequent_flyer/README.md

```diff
@@ -0,0 +1,61 @@
+# Gaia Frequent Flyer Example
+
+
+
+This example models a frequent flyer program.
+
+The schema tracks the following the information:
+
+- airport: Airport information.
+- airplane: Information about the airplane.
+- route: Information about route.
+- flight: Information about the flight. A flight is connected in a:
+  - 1:n relationship to travelers and segments.  
+  - 1:1 relationship with depart and arrival airports.
+- segment: Describes the trip segment. A segments is connected in a:
+  - 1:1 relationship to trips
+  - 1:1 relationship to flights
+- trip: Describes information about the itinerary for a single traveler. A trip is connected in a:
+  - 1:1 relationship to airport for arriving flights
+  - 1:1 relationship to airport for departing flights
+- traveler: Information about the traveler. Travelers are connected to trip
+
+The initial example defines three rules that use the traveler and flight tables
+
+After you understand these rules you can extend the sample here in the sandbox.
+
+## Rule 1
+
+Tracks lifetime miles for the traveler.
+
+- `flight_miles` from the flights table is defined as an [Active Field](https://gaia-platform.github.io/gaia-platform-docs.io/articles/rulesets-gaia-programming-model.html?q=active%20fields).
+- When the Frequent Flyer app updates the field in the database, the rules engine fires the rule.
+- The lifetime_miles is an implicit path flight->segments->trip->traveler
+- The Declarative C++ statement loops over each of the travelers on the flight and updates the lifetime_miles for each.
+
+## Rule 2
+
+Evaluates and updates the member level is for traveler. 
```

Comment:
I think there is an extra "is" here. Also, it should be "for a traveler", unless I'm missing the point of this description.